engine/src/main/java/org/terasology/engine/input/lwjgl/LwjglMouseDevice.java

```diff
@@ -123,6 +124,26 @@ public void propertyChange(PropertyChangeEvent evt) {
         }
     }
 
+    private void focusCallback(long window, boolean isFocused) {
+        // When the window is unfocused and focused again, the mouse coordinates
+        // reported by GLFW change even if the mouse hasn't moved.
+        // So when the window regains focus, we set the new position as the reference without generating deltas.
+        if (isFocused) {
+            DoubleBuffer mouseX = BufferUtils.createDoubleBuffer(1);
+            DoubleBuffer mouseY = BufferUtils.createDoubleBuffer(1);
+
+            GLFW.glfwGetCursorPos(window, mouseX, mouseY);
+            mouseX.rewind();
+            mouseY.rewind();
+
+            double x = mouseX.get(0);
+            double y = mouseY.get(0);
+    
+            this.xpos = x;
+            this.ypos = y;
```

Comment:
suggestion: I think these are short enough that you could `xpos = mouseX.get(0)` without the intermediate local variable.